src/stan/variational/advi_params_fullrank.hpp

```diff
@@ -0,0 +1,112 @@
+#ifndef STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
+#define STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
```

Comment:
Those were sort of notes to self.

The math library is closer to being stand alone. A lot of the folder level
includes have been removed.
On Jun 10, 2015 5:51 PM, "Alp Kucukelbir" <notifications@github.com> wrote:

> In src/stan/variational/advi_params_fullrank.hpp
> <https://github.com/stan-dev/stan/pull/1421#discussion_r32170832>:
>
> > @@ -0,0 +1,112 @@
> > +#ifndef STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
> > +#define STAN_VARIATIONAL_ADVI_PARAMS_FULLRANK__HPP
> > +
> > +#include <stan/math/prim/mat/fun/Eigen.hpp>
>
> i'm sorry, i didn't exactly follow what to do dan.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1421/files#r32170832>.
>